redaxo/src/addons/mediapool/lib/perm_helper.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * @package redaxo\mediapool
+ */
+class media_category_perm_helper
```

Comment:
Wir beginnen hier im Repo die Klassen immer mit `rex_`.

```suggestion
class rex_media_category_perm_helper
```

Reply:
82cd3f836ec190dca805e5a5cc154f9d218f3763